[DOCS-8804] Synthetics Advanced options: adding exit on success (#25078)
DataDog/documentation: content/en/synthetics/browser_tests/advanced_options.md

{{< img src="synthetics/browser_tests/advanced_options/timeout.png" alt="Timeout" style="width:25%">}}

## Exit on success

> Configure this option to exit the test after a successful step completion to avoid running unnecessary steps without needing to mark the test a failure.

Reviewer: ```suggestion
Configure this option to exit the test after a successful step completion. This prevents running unnecessary steps and avoids marking the test as a failure.
```

Author: Thanks that does sound better :)
